src/main/java/frc/robot/subsystems/Climber_Subsystem.java

```diff
@@ -27,29 +28,100 @@
     private WPI_TalonSRX horizontalL = new WPI_TalonSRX(RobotMap.horizontalClimb_L);
     private AHRS ahrs = new AHRS(SPI.Port.kMXP);
     private double vp = 0.05;
-    private double hp = 0.8;
+    private double hp = 0;//0.8;
     private double vError, vResponce, hError, hResponce = 0;
 
+    private Boolean updateState = null; 
+
     private int oldLVertical, oldRVertical, oldLHorizontal, oldRHorizontal = 0;
 
     public Climber_Subsystem() {
         super();
-        verticalR.configSelectedFeedbackSensor(FeedbackDevice.CTRE_MagEncoder_Relative, 0, 10);
-        verticalL.configSelectedFeedbackSensor(FeedbackDevice.CTRE_MagEncoder_Relative, 0, 10);
-        horizontalR.configSelectedFeedbackSensor(FeedbackDevice.CTRE_MagEncoder_Relative, 0, 10);
-        horizontalL.configSelectedFeedbackSensor(FeedbackDevice.CTRE_MagEncoder_Relative, 0, 10);
+        verticalR.configSelectedFeedbackSensor(FeedbackDevice.CTRE_MagEncoder_Relative, 0, 50);
+        verticalL.configSelectedFeedbackSensor(FeedbackDevice.CTRE_MagEncoder_Relative, 0, 50);
+        horizontalR.configSelectedFeedbackSensor(FeedbackDevice.CTRE_MagEncoder_Relative, 0, 50);
+        horizontalL.configSelectedFeedbackSensor(FeedbackDevice.CTRE_MagEncoder_Relative, 0, 50);
+
+    }
+    
+    //Initialization function that should be called only once
+    //This should be called before climbing but not on robot boot so manual adjustments to the climber
+    //before the game starts won't mess it up
+    public void singleInit()
+    {
+        //Hard reset the encoders to 0
+        verticalR.setSelectedSensorPosition(0, 0, 50);
+        verticalL.setSelectedSensorPosition(0, 0, 50);
+        horizontalR.setSelectedSensorPosition(0, 0, 50);
+        horizontalL.setSelectedSensorPosition(0, 0, 50);
+
+        // NOTE: Forward soft limit _must_ be the higher number. It doesn't really matter about which direction it goes
+        // soft limits are a range. Reverse is the low number and forward is the high number
+        int horizontalReverseLimit = 100;
```

Comment:
These are guesses, the vertical ones are based on Gabe's testing. The horizontal ones are based on nothing.